core/src/test/java/brooklyn/entity/rebind/RebindFeedTest.java

```diff
@@ -180,7 +180,7 @@ public void testSshFeedRegisteredInStartIsPersisted() throws Exception {
         EntityTestUtils.assertAttributeEqualsEventually(newEntity, SENSOR_INT, (Integer)0);
     }
 
-    @Test
+    @Test(groups="WIP") // intermittent failures
```

Comment:
Can we mark it WIP only in the 0.7.0 release branch ?

Would mean splitting up this PR.